src/index.html

```diff
@@ -11,7 +11,99 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset class="formBlock">
+        <legend>Personal information</legend>
+        <label class="classForm">Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="classForm">Name:
+          <input type="text" name="name">
+        </label>
+        <label class="classForm">How old are You?
+          <input
+            value="12"
+            type="number"
+            name="birth"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="classForm">Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label>I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>Registration</legend>
+        <label class="classForm">E-mail:
+          <input
+            placeholder="email@example.com"
+            type="email"
+            name="email"
+            required
+          >
+        </label>
+        <label>Password:
+          <input type="password" name="pass">
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>An interesting fact about you!</legend>
+        <div class="classForm">Do you love cats?
+          <label>
+            <input type="radio" name="cats"> Yes
+          </label>
+          <label>
+            <input type="radio" name="cats"> No
+          </label>
+        </div>
+        <label class="classForm">What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="classForm">What time do you go to bed?
+          <input type="time" name="bed">
+        </label>
+        <label class="classForm">What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Ladia">Lada</option>
+          </select>
+        </label>
+        <label>How do you rate our work?
+          <input type="range" name="rate">
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>Additional info:</legend>
+        <label class="classForm">Comments
+          <textarea
+            autocomplete="off"
+            name="comments"
+            minlength="10"
+            maxlength="100">
+          </textarea>
```

Comment:
```suggestion
          <textarea
            autocomplete="off"
            name="comments"
            minlength="10"
            maxlength="100"
          ></textarea>
```
To avoid 
![image](https://user-images.githubusercontent.com/97547824/203618095-55ed5e4f-81db-4a5b-80f7-cbe0550f98ed.png)